testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationInvitationLinkTest.java

```diff
@@ -299,7 +299,7 @@ private void registerUser(OrganizationResource organization, String expectedEmai
     }
 
     private void acceptInvitation(OrganizationResource organization, UserRepresentation user) throws MessagingException, IOException {
-        acceptInvitation(organization, user, "Account Management");
+        acceptInvitation(organization, user, "Sign in to");
```

Comment:
Does this test still work as expected? Seems like this now ends up on the login page instead.

Reply:
That was the page title when rendering-first the account console even though you have a redirect to the login page.

Reply:
Sounds good then, consider this resolved.